README.md

```diff
@@ -243,7 +243,7 @@ _path/to/myapp/linkerd.yml_, you could start linkerd in docker with
 the following command:
 
 ```
-$ docker run -p 4140:4140 -v path/to/myapp:/myapp -w /myapp buoyantio/linkerd:0.0.10-SNAPSHOT linkerd.yml
+$ docker run -p 4140:4140 -v /absolute/path/to/myapp:/myapp -w /myapp buoyantio/linkerd:0.0.10-SNAPSHOT linkerd.yml
```

Comment:
This is a big weakness in Docker(and I think the same in most container) volume mounting is that you can only mount a directory, not specific files. 
I do like getting rid of the -w and using absolute path, and it's what I did when I tried to run the example. I'll make that change.